classes/retro-core-image.bbclass

```diff
@@ -11,6 +11,7 @@ IMAGE_INSTALL += " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'retroarch-automount', 'udev-extraconf', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'retroarch-autostart systemd', 'retroarch-service', '', d)} \
     ${@bb.utils.contains('DISTRO_FEATURES', 'retroarch-firmware', 'firmware-libretro', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'resize-helper', 'resize-helper', '', d)} \
```

Comment:
`${@bb.utils.filter('DISTRO_FEATURES', 'kodi rauc resize-helper polkit', d)}` should also work.